### DISCUSSION TITLE
ZMAI-90 and tasmota 9.2.0

### DISCUSSION THREAD

**Discussion by nonyhaha** [arendst/Tasmota#10755] [Q&A] [answered]
Hello everybody! 
I have been using the zmai 90 power meter for a long time, and because I was running an older version of Tasmota - 8.3 I think, I wanted to upgrade it. The problem is that, after the upgrade, it is not working anymore. 
I am using the same generic(0) config with the same pins I was using on tasmota 8.3. , as mentioned in the instructions here https://www.creationfactory.co/2019/12/reverse-engineering-zmai-90-din-rail.html . 
I think the issue is with the choosing of the gpio pins or the module type, as, on the main page, I only get the switch option, and none of the values I used to get before, like voltage, all power values.... 

Does anybody know if anything changed between 8.3 and 9.2? should I use other pins for rx/tx, or select anything else in the dropdown menus?

**Comment by sfromis** [comment, marked as answer]
I've got no experience with that device, but there are some setup instructions in the template entry for it.
https://templates.blakadder.com/ZMAi-90

**Reply by nonyhaha** [reply]
Helli. I followed those instructions, but none of the variants work. There is an issue with moving from tasmota 8.3 to 9.2. it used to work before this using those instructions.
